### DISCUSSION TITLE
Please add these features

### DISCUSSION THREAD

**Discussion by Savemypeanuts** [YouTube-Enhancer/extension#580] [Ideas]
Automatically skip intros, or a button to do that, like Netflix does. Detection for sponsored portions of the video, where a YouTuber promotes a product, automatically skip this without clicking a button, because nobody wants to watch that.

**Comment by VampireChicken12** [comment]
Thanks for the suggestions. I will look into implementing those in the meantime can you make an issue for each of the features you have suggested. @Savemypeanuts

**Comment by Savemypeanuts** [comment]
How do I do that, sir? Lol that tag, why did I call myself that

**Comment by Savemypeanuts** [comment]
Oh so I might as well tell you about the Android app ReVanced. It has
showed me how YouTube premium should be. Please use it as a reference, I'd
love to use YouTube on desktop the same way I use it on my phone.

**Reply by VampireChicken12** [reply]
> Oh so I might as well tell you about the Android app ReVanced. It has showed me how YouTube premium should be. Please use it as a reference, I'd love to use YouTube on desktop the same way I use it on my phone.
> […](#)
> On Fri, 16 Aug 2024, 21:03 Bob Blank, ***@***.***> wrote: How do I do that, sir? Lol that tag, why did I call myself that On Fri, 16 Aug 2024, 20:56 Nathan, ***@***.***> wrote: > Thanks for the suggestions. I will look into implementing those in the > meantime can you make an issue for each of the features you have suggested. > @Savemypeanuts <https://github.com/Savemypeanuts> > > — > Reply to this email directly, view it on GitHub > <[#580 (comment)](https://github.com/YouTube-Enhancer/extension/discussions/580#discussioncomment-10362519)>, > or unsubscribe > <https://github.com/notifications/unsubscribe-auth/BHQPF6XTVL6CY4LL6W3VOXDZRZKNLAVCNFSM6AAAAABMUT6AVOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZWGI2TCOI> > . > You are receiving this because you were mentioned.Message ID: > ***@***.*** > .com> >

I am aware of ReVanced what do you want me to implement based on it

**Comment by Savemypeanuts** [comment]
Well the skip intro has stuck out to me and the automatic sponsor skip was
one of those things I never thought I wanted until I had it. Alternative
thumbnails might be a nifty feature for some other people, not a priority
for me, I can appreciate a YouTuber putting time into an intentional
eye-grab. Does your project enable downloads? That would be sick. I truly
appreciate that you help me stay off shorts by the way, they have become a
very bad habit of mine, literally rot the brain.

So to easily sum it up:

Skip intro button
Automatically skip sponsored segments
Downloads
Alternative thumbnails (but it won't go amiss if you don't incorporate this
one)

**Comment by VampireChicken12** [comment]
thank you for your reply I'll look into those things

**Comment by Savemypeanuts** [comment]
Appreciate men like you, bro. Smarter than me, all I can do is request and
hope. I greatly appreciate you being on this earth, you mean a lot.

**Reply by VampireChicken12** [reply]
thank you for your kind words

**Comment by rlwings** [comment]


**Reply by VampireChicken12** [reply]
I think you are confusing my extension with another. As far as I am aware the features work fine. @rlwings

**Comment by Savemypeanuts** [comment]
Still works well for me. Automatic theatre mode is a blessing 🙏🏻 Among
the many, many other features.

**Reply by VampireChicken12** [reply]
> Still works well for me. Automatic theatre mode is a blessing 🙏🏻 Among the many, many other features.
> […](#)
> On Sun, 10 Nov 2024, 23:43 Nathan, ***@***.***> wrote: I think you are confusing my extension with another. As far as I am aware the features work fine. @rlwings <https://github.com/rlwings> — Reply to this email directly, view it on GitHub <[#580 (reply in thread)](https://github.com/YouTube-Enhancer/extension/discussions/580#discussioncomment-11208850)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BHQPF6VGO5QPSJ45DEPX4WTZ77VQRAVCNFSM6AAAAABMUT6AVOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMRQHA4DKMA> . You are receiving this because you were mentioned.Message ID: ***@***.*** com>

thanks for replying

**Comment by DarkSky7** [comment]
Could you make it possible to change font-family?  I'd love to use a serif font, like Anko, Artifex, ... https://designshack.net/articles/inspiration/fonts-similar-to-times-new-roman/

**Reply by VampireChicken12** [reply]
> Could you make it possible to change font-family? I'd love to use a serif font, like Anko, Artifex, ... [designshack.net/articles/inspiration/fonts-similar-to-times-new-roman](https://designshack.net/articles/inspiration/fonts-similar-to-times-new-roman/)

I can look into this. It is easier if you make a issue requesting a feature. I forget about the stuff in these discussions 🤣

**Comment by DarkSky7** [comment]
It would be cool if the view pane popped out/hovered when you scrolled the window down to look at comments further down the page.

**Reply by VampireChicken12** [reply]
> It would be cool if the view pane popped out/hovered when you scrolled the window down to look at comments further down the page.

I plan on working on a mini player feature eventually not sure when that will happen.

**Comment by DarkSky7** [comment]
In very dark modes, the comment like button disappears into the background.  Please consider options that might improve that.

**Reply by VampireChicken12** [reply]
> In very dark modes, the comment like button disappears into the background. Please consider options that might improve that.

Is this when using YouTube DeepDark themes? I have no direct control over them I just added support for them in my extension instead of the user having to have stylus and the theme itself. A lot of people like to cut down on extensions in their browser.

**Comment by DissenterNet** [comment]
I discovered this extension looking for a feature that isn't included but I installed the extension anyways because it looks great. The feature that I would like to see added if you get a chance is to block members only content from showing up in the feed or if that is tough maybe just make it easy to tell what is members only and skim past it. It is very annoying seeing stuff that looks interesting then clicking and getting the members only crap. Id love to just not see any of that because I will never be a member of any of it.

Either way thanks for your gift of this extension. much appreciated!

**Reply by DissenterNet** [reply]
Okay my bad, I wrote this based on the screenshots in the extension page, when I went into the settings I see that you already added this feature. That is awesome thanks! this extension did everything I wanted. If you have extra money give this projects some for me please!

**Reply by VampireChicken12** [reply]
@DissenterNet  It's okay I need to update those screenshots but I've been busy. Thank you for using the extension
